src/frontend/http.rs

```diff
@@ -496,7 +500,32 @@ pub fn filters(
         .allow_methods(vec!["GET", "POST"])
         .max_age(CORS_MAXAGE);
 
-    let log = warp::log(module_path!());
+    let log = warp::log::custom(|info: Info<'_>| {
+        let duration = format!("{:?}", info.elapsed());
+
+        #[cfg(feature = "metrics")]
+        counter!(
+            HTTP_REQUESTS,
+            "method" => info.method().as_str().to_string(),
+            "path" => info.path().to_string(),
+            "status" => info.status().as_u16().to_string(),
+            //"duration" => duration.clone(),
```

Comment:
I think it's ok to include this, just a note that it will result in separate metric counts (the metrics are grouped by all fields)